src/Commands/Add.php

```diff
@@ -0,0 +1,110 @@
+<?php
+
+namespace Larowlan\Tl\Commands;
+
+use Larowlan\Tl\Connector\Connector;
+use Larowlan\Tl\Connector\ConnectorManager;
+use Larowlan\Tl\Formatter;
+use Larowlan\Tl\Repository\Repository;
+use Stecman\Component\Symfony\Console\BashCompletion\Completion\CompletionAwareInterface;
+use Stecman\Component\Symfony\Console\BashCompletion\CompletionContext;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ *
+ */
+class Add extends Command implements LogAwareCommand {
+
+  /**
+   * @var \Larowlan\Tl\Connector\Connector
+   */
+  protected $connector;
+
+  /**
+   * @var \Larowlan\Tl\Repository\Repository
+   */
+  protected $repository;
+
+  /**
+   *
+   */
+  public function __construct(ConnectorManager $connector, Repository $repository) {
+    $this->connector = $connector;
+    $this->repository = $repository;
+    parent::__construct();
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function configure() {
+    $this
+      ->setName('add')
+      ->setDescription('Add a time entry')
+      ->setHelp('Add a new entry, for a given that. <comment>Usage:</comment> <info>tl add [ticket number] [duration in hours]</info>')
+      ->addArgument('issue_number', InputArgument::REQUIRED, 'Issue number to start work on')
+      ->addArgument('duration', InputArgument::REQUIRED, 'Duration in hours to change slot to')
+      ->addArgument('comment', InputArgument::OPTIONAL, 'Comment to start with')
+      ->addOption('start', 's', InputOption::VALUE_OPTIONAL, 'A date offset', 'now')
```

Comment:
Description needs re-work. Should be something like an _absolute or relative date_.